client/components/foldable-card/style.scss

```diff
@@ -135,6 +136,20 @@ button.foldable-card__action {
 	display: none;
 }
 
+.foldable-card.is-smooth .foldable-card__content {
+	display: block;
+	overflow: hidden;
+
+	transition: max-height 0.25s cubic-bezier(0.175, 0.885, 0.32, 1.275), padding 0.2s ease-in;
```

Comment:
We shouldn't be transitioning the `padding` here, we only want to transition `max-height` (actually there shouldn't be any padding on this transitioning container in the first place (see explanation below).